packages/ag-charts-community/src/module/optionsGraph.ts

```diff
@@ -305,7 +312,10 @@ export class OptionsGraph extends AdjacencyListGraph<unknown, string> implements
         // the original object and must be found in the graph.
         const pathVertex = this.findVertexAtPath(path);
         if (pathVertex) {
-            return this.findNeighbour(pathVertex, USER_OPTIONS_EDGE) != null;
+            return (
+                this.findNeighbour(pathVertex, USER_OPTIONS_EDGE) != null ||
+                this.findNeighbour(pathVertex, USER_PARTIAL_OPTIONS_EDGE) != null
+            );
```

Comment:
As discussed - this bug should be fixed by the similar changes made here https://github.com/ag-grid/ag-charts/pull/4959/files#diff-27b8479ea6fa70d9276fa0fb48c97885b34220e6c9169f5b4fafb808d98b4757R414-R417